code/modules/food_and_drinks/food/candy.dm

```diff
@@ -134,14 +128,15 @@
 	trash = /obj/item/trash/candy
 	filling_color = "#7D5F46"
 	bitesize = 3
-	list_reagents = list("nutriment" = 2, "chocolate" = 5)
+	junkiness = 25
```

Comment:
Honestly, I still think that any candy should be considered "junk food". It's mostly sugar anyways, which doesn't exactly keep you going for long.